src/components/Chat/Chat.tsx

```diff
@@ -162,9 +176,16 @@ const ChatWithContext = <
     Us
   >(client);
 
+  useAppStateListener<At, Ch, Co, Ev, Me, Re, Us>(
+    client,
+    closeConnectionOnBackground,
+  );
+
   useEffect(() => {
     if (client.setUserAgent) {
       client.setUserAgent(`stream-chat-react-native-${Platform.OS}-${version}`);
```

Comment:
There will be a merge conflict with the {version} change just merged heads up.

Reply:
Will be fine I think. There is no diff here on this PR. Github would have raised it by now. Will pull master anyways, when I fix the `Chat.js` comment.

Reply:
Done!! Synced with master!!